server/testUtils/booleanUtils.ts

```diff
@@ -0,0 +1,15 @@
+export const generateBooleanCombinations = (n: number) => {
```

Comment:
Could you add a function definition to this? It doesn't need types since we're using TypeScript, but it would be good to have a description and usage example e.g. 
```
/**
 * Generates boolean combinations of a specified length
 *
 * @param n - The length of each boolean combination to generate
 * @returns An array of boolean arrays, each intended to be length `n`
 * 
 * @example
 * generateBooleanCombinations(2)
 * // Might return:
 * // [
 * //   [true, true],
 * //   [true, false],
 * //   [false, true]
 * // ] 

```